File: Go/Euler019.go
package main

import (
	"fmt"
	"time"
)

func main() {
	const day = 1

	var sundays int

	for year := 1901; year <= 2000; year++ {
		for month := 1; month <= 12; month++ {
			date := time.Date(year, time.Month(month), 1, 0, 0, 0, 0, time.UTC)
			if date.Weekday() == time.Sunday {
				sundays++
			}
		}
	}

	fmt.Println(sundays)
}

File: Python/Euler019.py
#!/usr/bin/env python3
from __future__ import print_function
from datetime import datetime
import sys
if sys.version.startswith('2'):
    range = xrange


def main():
    sunday = 6
    sundays = 0

    for year in range(1901, 2001):
        for month in range(1, 13):
            d = datetime(year, month, 1)
            if d.weekday() == sunday:
                sundays += 1

    print(sundays)


if __name__ == '__main__':
    main()
